modules/sno-du-subscribing-to-the-operators-needed-for-platform-configuration.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-subscribing-to-the-operators-needed-for-platform-configuration_{context}"]
+= Subscribing to the Operators
+
+The subscription provides the location to download the Operators needed for platform configuration.
+
+.Procedure
+
+* Use the following example to configure the subscription:
```

Comment:
Do these features need to be installed? cluster logging, local storage, performance add-on operator, etc?

Reply:
It appears so. @imiller0 Clarification?

Reply:
These subscriptions result in the Cluster Logging, Local Storage Operator, Performance Addon Operator, PTP, and SRIOV operators being installed.